README.md

```diff
@@ -211,24 +217,23 @@ The function can also be async:
 // lint-staged.config.js
 const micromatch = require('micromatch')
 
-module.exports = (allStagedFiles) => {
-    const shFiles =  micromatch(allStagedFiles, ['**/src/**/*.sh']);
-    if (shFiles.length) {
-      return `printf '%s\n' "Script files aren't allowed in src directory" >&2`
-    }
-    const codeFiles = micromatch(allStagedFiles, ['**/*.js', '**/*.ts']);
-    const docFiles = micromatch(allStagedFiles, ['**/*.md']);
-    return [`eslint ${codeFiles.join(' ')}`, `mdl ${docFiles.join(' ')}`];
+module.exports = allStagedFiles => {
```

Comment:
It seems you ran prettier on the README.md file with incorrect settings; maybe wrong version?

Reply:
Whoops, my editor autoformatted them. That's my bad, I'll revert that